inference-engine/src/inference_engine/src/cpp/async_infer_queue.cpp

```diff
@@ -0,0 +1,135 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/runtime/async_infer_queue.hpp"
+
+#include <condition_variable>
+#include <map>
+#include <mutex>
+#include <queue>
+#include <string>
+#include <vector>
+
+#include "ie/ie_plugin_config.hpp"
+
+namespace ov {
+namespace runtime {
+
+AsyncInferQueue::AsyncInferQueue(ov::runtime::ExecutableNetwork net, size_t jobs) {
+    // Automatically set number of jobs
+    if (jobs <= 0) {
```

Comment:
`size_t` is unsigned so it can't be less than 0. Is there any other use case that this comparison handles?

Reply:
I overlooked that check as is was previously `int64_t`, thanks!